flake.nix

```diff
@@ -58,6 +58,7 @@
       framework = import ./framework;
       framework-12th-gen-intel = import ./framework/12th-gen-intel;
       friendlyarm-nanopc-t4 = import ./friendlyarm/nanopc-t4;
+      focus-m2-gen1 = import ./focus/m2/gen1;
```

Comment:
Please also add in the README because this is what we actually use in CI for discovering modules.

Reply:
My bad, fixed :white_check_mark: